### ISSUE TITLE
“Changes updated” message is not apparent to TalkBack users [Android]

### ISSUE THREAD

**Issue by jonnyjames** [covid-alert-ny/covid-green-app#296]
Upon confirming changes on the App Settings -> My Health Log screen, the status message “Changes updated” is displayed at the top of the main content of the screen. It is not immediately apparent to TalkBack users, as it is not announced. Focus is moved to the top of the screen after form submission. The “COVID ALERT NY” announcement may lead TalkBack users to believe they have been taken to a new screen.

![settings - Changes updated](https://user-images.githubusercontent.com/7034772/92129487-96aba300-edd1-11ea-877a-0ee9e9ff9cbc.png)

We suggest moving focus to the message upon its appearance.

**Comment by floridemai** [comment]
@jonnyjames This should be fixed in the next build.

**Comment by jonnyjames** [comment]
@floridemai This issue is now fixed. Closing this issue.
